src/utils.js

```diff
@@ -7,15 +7,14 @@ const API_CONFIG = {
     live: 'https://niftyfootball.cards/api'
 };
 
-const getApi = () => {
-    switch (window.location.hostname) {
-        // case "localhost":
-        // case "127.0.0.1":
-        //     return API_CONFIG.local;
-        default:
-            return API_CONFIG.live;
-    }
-};
+const live = window.location.hostname !== "localhost" && window.location.hostname !== "127.0.0.1"
+
+// if live use mainnet, otherwise use rinkeby
+const DEV_NETWORK_ID = 4;
+const PROD_NETWORK_ID = 1;
+const NETWORK_ID = live ? PROD_NETWORK_ID : DEV_NETWORK_ID;
```

Comment:
I think we will tweak this once its in as we'd like to test on live from local as well, no biggy we can change this